routing/router.go

```diff
@@ -1639,98 +1719,288 @@ func (r *ChannelRouter) sendPayment(payment *LightningPayment,
 
 	timeoutChan := time.After(payAttemptTimeout)
```

Comment:
After restart, we shouldn't restart the payment attempt timer. Alternative: store a wall clock time at which the payment attempts will stop?

Reply:
This is intentionally left for follow-up PR.

Reply:
Add todo comment?

Reply:
Added todo: https://github.com/lightningnetwork/lnd/pull/2761/commits/12fa43c6240b6282713d4e416998cc6faa7f474d